docs/plugins/README.md

```diff
@@ -77,24 +73,31 @@ export default {
 ```
 
 ## Plugin API
-
-Plugins at the end of the day, are just files (or a `directory` that contains the files):
-
+All plugins contains two at least files:
 - `node.api.js` - Exposes the [Node Plugin API](/docs/plugins/node-api.md)
 - `browser.api.js` - Exposes the [Browser Plugin API](/docs/plugins/browser-api.md)
 
-## Why separate node and browser entry points for plugins?
+## Why use separate node and browser entry points for plugins?
 
-We use separate entry points for node and browser context so as to not create conflict with imports that may not be supported in both environments.
+Separating plugin entrypoints avoids creating conflict with imported modules that may not be supported in both environments.
 
-To use either API, the corresponding API file must:
+## How to write the `node.api.js` and `browser.api.js` files when creating plugins?
+The file (for either environment) must:
 
 - Provide a `function` as the `default export`
-- That function receives **plugin options from the user (optional)**
+- That function receives an optional **user plugin options** argument
 - **Return an `object`** providing any **API methods** to implement
 
-Here is a **pseudo** example of what a plugin typically looks like:
+E.g. of basic plugin:
+```javascript
+export default pluginOptions => ({
+  myMethod: options => {
+    console.log('hello world');
+  }
+})
 
+```
+E.g. of more complex plugin
```

Comment:
Do we need to examples?

Reply:
guess not. seems redundant. I've consolidated both

Reply:
👍